addon/mixins/touch-action.js

```diff
@@ -37,15 +36,15 @@ export default Mixin.create({
   ignoreTouchAction: false,
 
   init() {
-    this._super();
+    this._super(...arguments);
 
     const {
       tagName,
       ignoreTouchAction,
       click
     } = this;
 
-    const hasClick = click !== K;
+    const hasClick = typeof click === 'function';
```

Comment:
typeof checks are risky, they often deopt and even when they don't they almost always cause Chrome to never optimize a loop. Given this is a check within init of every component, I'd prefer to avoid it, so my intent was that we explicitly set `click` to `K`, allowing us to do a dirty reference check `click !== K` that simply informs us whether the end user has set `click` to something on their own. I'm alright with it erring out if they've explicitly set it to something that's not a function, because that would actually be a mistake on their part.